src/families/filecoin/bridge/account.ts

```diff
@@ -0,0 +1,307 @@
+import {
+  AmountRequired,
+  FeeNotLoaded,
+  InvalidAddress,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  NotEnoughBalance,
+  RecipientRequired,
+} from "@ledgerhq/errors";
+import { BigNumber } from "bignumber.js";
+import Fil from "@zondax/ledger-filecoin";
+import { Observable } from "rxjs";
+
+import { makeAccountBridgeReceive, makeSync } from "../../../bridge/jsHelpers";
+import {
+  Account,
+  AccountBridge,
+  AccountLike,
+  BroadcastFnSignature,
+  SignOperationEvent,
+  SignOperationFnSignature,
+  TransactionStatus,
+} from "../../../types";
+import { Transaction } from "../types";
+import { getAccountShape, getAddress, getTxToBroadcast } from "./utils/utils";
+import { broadcastTx, fetchBalances, fetchEstimatedFees } from "./utils/api";
+import { getMainAccount } from "../../../account";
+import { close } from "../../../hw";
+import { toCBOR } from "./utils/serializer";
+import { Operation } from "../../../types/operation";
+import { calculateEstimatedFees, getPath, isError } from "../utils";
+import { log } from "@ledgerhq/logs";
+import { getAddressRaw, validateAddress } from "./utils/addresses";
+import { patchOperationWithHash } from "../../../operation";
+import { withDevice } from "../../../hw/deviceAccess";
+
+const receive = makeAccountBridgeReceive();
+
+const createTransaction = (): Transaction => {
+  log("debug", "[createTransaction] creating base tx");
+
+  return {
+    family: "filecoin",
+    amount: new BigNumber(0),
+    method: 0,
+    version: 0,
+    nonce: 0,
+    gasLimit: new BigNumber(0),
+    gasFeeCap: new BigNumber(0),
+    gasPremium: new BigNumber(0),
+    recipient: "",
+    useAllAmount: false,
+  };
+};
+
+const updateTransaction = (t: Transaction, patch: Transaction): Transaction => {
+  log("debug", "[updateTransaction] patching tx");
```

Comment:
please remove some of the tracing logs debug on this file, was probably interesting during development but it's better we try to keep it concise. (typically no need to log on hooks because we look it in different ways in LLD / no need to log "loading fees" because we see a log for the network call, etc..)

Reply:
I have commented all log lines of that kind. I think they are useful for debugging, so I left them commented so that I can use them again if we have to make some fix.